behind-the-veil-siteroot/imports/ui/components/map/MarkerMap.jsx

```diff
@@ -56,7 +144,7 @@ export const MarkerMap = ({ className, location }) => {
         map.current.remove();
       }
     };
-  }, [location, australiaBounds]);
+  }, [location, radius, australiaBounds]);
```

Comment:
Small thing. I'm not sure if location or radius are states, but australiaBounds is definitely just a constant, so it doesn't need to be included in the dependency array b/c it will never change